js/bit2c.js

```diff
@@ -389,6 +389,9 @@ module.exports = class bit2c extends Exchange {
             'Amount': amount,
             'Pair': market['id'],
         };
+        if ('Total' in params) {
```

Comment:
This is not needed since the request is extended with params here: https://github.com/ccxt/ccxt/pull/14594/files#diff-ed9560d51f98ffe85c6348fb5eeb35a3b48573bc854351187fa3818904243996R402